tests/test_cases/test_vhdl_zerovector/test_vhdl_zerovector.py

```diff
@@ -0,0 +1,49 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import cocotb
+from cocotb.result import TestFailure
+from cocotb.triggers import Timer
+
+
+@cocotb.test()
```

Comment:
Sorry for lots of single comments, one at a time... this one is the last hopefully :)
```suggestion
@cocotb.test(expect_error=AttributeError if cocotb.SIM_NAME.lower().startswith('ghdl') else ())
```
GHDL doesn't expose the zero-length vector. We allow GHDL CI to fail so one more error is not critical, but while we are here, we can make it explicit.

Reply:
Should be fixed by 87e74776aaabc489fa09b8039685633b496acc08

Reply:
Race condition :) I had added `"xmsim", "ncsim"` to the list while you were making the change of my initial version.
Beware, I have just now made yet *another* fix to add an additional set of parentheses.

(And thanks for spotting that this is not the right location to expect an error.)

Reply:
Thank you for your help. I updated the code accordingly (e557313d70299b576b1548f0854436557d1f2ffb)